worker/build.sbt

```diff
@@ -6,56 +6,32 @@ version := "1.11"
 
 scalaVersion := "2.11.7"
 
-crossScalaVersions := Seq(scalaVersion.value, "2.11.6")
+crossScalaVersions += "2.10.5"
```

Comment:
Okay. It is true that 2.10 is EOL.

We can look forward to 2.12 in a few months :)